Tests/PointFreeTests/AtomFeedTests.swift

```diff
@@ -1,16 +1,50 @@
 import ApplicativeRouter
 import SnapshotTesting
+import Optics
 import Prelude
 import XCTest
 @testable import PointFree
 import PointFreeTestSupport
 import HttpPipeline
 
 class AtomFeedTests: TestCase {
+  override func setUp() {
+    super.setUp()
+//    record = true
+  }
+
   func testAtomFeed() {
     let conn = connection(from: request(to: .feed(.atom)))
     let result = conn |> siteMiddleware
 
     assertSnapshot(matching: result.perform())
   }
+
+  func testEpisodeFeed() {
+    let conn = connection(from: request(to: .feed(.episodes)))
+    let result = conn |> siteMiddleware
+
+    assertSnapshot(matching: result.perform())
+  }
+
+  func testEpisodeFeed_WithRecentlyFreeEpisode() {
```

Comment:
this test shows that an older episode that was recently made free gets bumped to the top